MVP Module/Source/Feed Presentation/FeedPresenter.swift

```diff
@@ -9,21 +9,28 @@ protocol FeedLoadingView {
 	func display(_ viewModel: FeedLoadingViewModel)
 }
 
+protocol FeedErrorView {
+	func display(_ viewModel: FeedErrorViewModel)
+}
+
 protocol FeedView {
 	func display(_ viewModel: FeedViewModel)
 }
 
+typealias FeedViewFlow = FeedView & FeedErrorView
+
 final class FeedPresenter {
-	private let feedView: FeedView
+	private let feedView: FeedViewFlow
 	private let loadingView: FeedLoadingView
```

Comment:
Instead of making the feedView also responsible for displaying errors, we can inject different objects to handle these responsibilities:
```suggestion
	private let feedView: FeedView
	private let loadingView: FeedLoadingView
	private let errorView: FeedErrorView
```

Reply:
done